csrankings-o.csv

```diff
@@ -102,7 +102,6 @@ Oliver Yang,University of Ottawa,http://www.site.uottawa.ca/~yang,NOSCHOLARPAGE
 Oliver van Kaick,Carleton University,http://people.scs.carleton.ca/~olivervankaick,IfvyBBUAAAAJ
 Olivier Alata,Université Jean Monnet,http://perso.univ-st-etienne.fr/ao29170h,ClqgTp8AAAAJ
 Olivier Caron,CRIStAL,https://www.cristal.univ-lille.fr/profil/carono,NOSCHOLARPAGE
-Olivier Danvy,Aarhus University,http://cs.au.dk/~danvy,OZJGoVoAAAAJ

```

Comment:
Please update affiliation and web page (https://www.yale-nus.edu.sg/faculty/olivier-danvy/).